base/applications/network/CMakeLists.txt

```diff
@@ -11,8 +11,10 @@ add_subdirectory(nslookup)
 add_subdirectory(ping)
 add_subdirectory(route)
 if(NOT USE_CLANG_CL)
+if(NOT ARCH STREQUAL "arm" AND NOT ARCH STREQUAL "arm64") # FIXME: CORE-17606
```

Comment:
Disabling building of modules is not the way to go, I'm against it.

Reply:
The current situation makes no sense to me:
Mark does not want to remove `-DENABLE_ROSTESTS=1 -DENABLE_ROSAPPS=1`, which are causing failures when trying to actually build them,
you do not want to make them succeed, as much as possible.
How to make progress with such "no no"s?

Reply:
Eventually removed the 2 commits which disabled applications, as the later have been fixed.